### ISSUE TITLE
Feature request: keep the position in the timeline

### ISSUE THREAD

**Issue by rbatllet** [sindresorhus/anatine#79]
I'd like to retain the last seen/read post position to avoid losing unread posts every day.

It will be possible it will be great!

**Comment by kevva** [comment]
Doesn't it already? See https://github.com/sindresorhus/anatine/issues/8.

**Comment by rbatllet** [comment]
It's nothing about the scroll. It refers to keep the last read post like when you read an ebook and get the last read page.

**Comment by rbatllet** [comment]
What about use the Tweet Marker (http://tweetmarker.net/) ?.

```Tweet Marker is an award-winning, cross-platform web service for syncing the reading position between multiple Twitter clients.```

**Comment by paulmolluzzo** [comment]
:-1: I'm not particularly interested in this idea and it seems antithetical to Twitter. The whole point of the firehose is it'll be up to the newest activity when I come back. What if I scroll down on Friday and don't open the app until Monday? I'll be 3 days behind?

**Comment by rbatllet** [comment]
I disagree @paulmolluzzo, It's important not to lose unread posts. It's vital to get the last reading position to avoid it.

**Comment by rbatllet** [comment]
@paulmolluzzo Can you imagine reading an ebook without knowing what page you read before?

**Comment by paulmolluzzo** [comment]
This is Twitter's default behavior. I follow a few hundred people and have no interest in reading every tweet since my last login to "catch up". Personally I would just scroll to the top every time.

This isn't an ebook, most tweets are standalone content that require little/no context or are in a reply thread, and, IMHO, fully 100% of Twitter is non-vital information. I don't care how many tweets I miss.

All that said, we can just agree to disagree. :smile: If other people want the feature and someone wants to do a PR go for it.

**Comment by rbatllet** [comment]
You can disable it if you want. If you don't read the posts of people you follow, why are you following them?. 

It is a bit inconsistent, isn't it?

**Comment by mafredri** [comment]
I'd also be interested in Tweet Marker support, so I thought I'd contribute my view and reasoning:

1. I read Twitter mostly on my phone because I dislike re-reading Tweets I've already read. Syncing read status between phone and desktop would make this a much more pleasant experience for me.
2. Scrolling up to the latest Tweet is my preferred method because it avoids 1. Whenever I scroll from top to bottom I end up reading old stuff I've already read (until I notice it).
3. If I find myself with a backlog of days I sometimes just `Goto most recent`, and maybe scroll down a bit to see what's up.
4. My personal preference would be to have both a `Goto most recent` and a `Goto last read`, but it's not strictly necessary.

**Comment by gingerbeardman** [comment]
I used to use Echofon for its ability to sync my read position across mobile and desktop. I stopped using it when I was following so many people I just couldn't keep up, that point will come eventually for most people.
